feature-libs/cart/saved-cart/styles/_saved-cart-list.scss

```diff
@@ -65,29 +79,30 @@ cx-saved-cart-list {
     }
 
     .cx-saved-cart-list-thead-mobile {
-      @include media-breakpoint-down(sm) {
+      @include media-breakpoint-down(md) {
         display: none;
       }
     }
 
     .cx-saved-cart-list-cart-name {
-      @include media-breakpoint-up(md) {
-        width: 15%;
-      }
+      padding-inline-start: 20px;
 
       @include media-breakpoint-up(lg) {
         width: 20%;
       }
 
       .cx-saved-cart-list-value {
         text-decoration: underline;
+        color: var(--cx-color-primary);
       }
     }
 
     .cx-saved-cart-list-make-cart-active {
       .cx-saved-cart-make-active {
         text-transform: uppercase;
```

Comment:
This seems to be overriding some Desing System standard values, afaik, we only have uppercase text in 3 places: top navigation, table headers and footer headings.
The saved cart list is a table, so it should follow along the same styles as e.g Order History and the same for action links, they should look like the screenshot below:

![Screen Shot 2021-03-15 at 3 04 14 PM](https://user-images.githubusercontent.com/15113219/111210877-29f53a80-85a4-11eb-964e-898355365344.png)

![Screen Shot 2021-03-15 at 3 28 58 PM](https://user-images.githubusercontent.com/15113219/111210911-35486600-85a4-11eb-8214-5d56854a93c2.png)